docs/developer-guide/ksql-testing-tool.rst

```diff
@@ -38,6 +38,26 @@ To test a set of KSQL statements, you provide three files, one file containing t
                 This option may occur a maximum of 1 time
 
 
+Provide required jar files
+**************************
+
+Starting in version 5.3.0, ksql-test-runner throws a
+``ClassNotFoundException``, because the the following jars aren't packaged in
+KSQL.
+
+- `ksql-functional-tests/hamcrest-all-1.3.jar`
+- `ksql-functional-tests/junit-4.12.jar`
+
+The workaround is to copy the required jars into the ``confluent/ksql``
```

Comment:
Is `confluent/ksql` the right path here?

Reply:
true, the path mentioned in https://github.com/confluentinc/ksql/issues/3387 is `confluent-5.3.1/share/java/ksql`